### PR TITLE
Adds basic discussion rules

### PR DESCRIPTION
**Pull request by Iamgoofball**
These should facilitate a happier dev environment.

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
even if it's a meme I like this pr

**Comment by Repukan** [comment]
Hey shitdick you forgot to put "don't insult the op."

**Comment by Iamgoofball** [comment]
Covered by 1C

**Comment by Repukan** [comment]
That's tinfoil shit not rudeness and manners.

**Comment by Repukan** [comment]
8======D

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -21,6 +21,16 @@ There is an open list of approachable issues for [your inspiration here](https:/
...
+
+1B. "I really don't like the lore behind this". This is a perfectly understandable sentiment, but it'd be much better suited for the forums. A feature can always be rethemed after being merged if it's that badly implemented. Getting the maintainers and coders distracted about lore can allow code or gameplay issues to slip through un-noticed under the fog of war over theme and lore. Another thing to keep in mind is we [merge](https://github.com/tgstation/tgstation/pull/22235) [theme-breaking](https://github.com/tgstation/tgstation/pull/22314) [references](https://github.com/tgstation/tgstation/pull/21488) all the time, and whatever is being added likely won't be worse than [this](https://github.com/tgstation/tgstation/pull/22115), and some people actually liked that PR for whatever godawful reason.
+
+1C. "FUCKING (((CODERBUS))) TRYING TO RUIN OUR SS13 WITH THEIR SJW HORRIBLE FEATURES WTF GODDAMNT IT FIGHT THE POWER". You posted this on the wrong feedback location. Please post these comments [here](https://www.singulo.io/tg/).
+
```
Comment (Shadowlight213) [inline .github/CONTRIBUTING.md]: 1D. Don't be Iamgoofball.  Being Iamgoofball will cause all previous rules to be null and void when others are referring to you. Additionally, Being Iamgoofball will cause your PR to automatically be closed.

**Pull request closed without merging**